guides/common/modules/proc_renewing-a-custom-ssl-certificate-on-smart-proxy.adoc

```diff
@@ -0,0 +1,75 @@
+You must keep the {SmartProxyServer} SSL certificates on the {ProjectServer} by creating a folder and verifying them through the `katello-certs-check` command.
+
+.Procedure
+. On {ProjectServer}, validate the {customssl} certificate input files:
++
+[options="nowrap" subs="+quotes,attributes"]
+----
+# katello-certs-check \
+-c _/root/{smart-proxy-context}_cert/New_{smart-proxy-context}_cert.pem_ \
+-k _/root/{smart-proxy-context}_cert/{smart-proxy-context}_cert_key.pem_ \
+-b _/root/{smart-proxy-context}_cert/ca_cert_bundle.pem_
+----
++
+If the CN has a wildcard value, you must add the `-t {smart-proxy-context}` option to the `katello-certs-check` command.
+. Use the following `{certs-generate}` commands to generate the certificate archive file for your {SmartProxyServer}:
++
+[options="nowrap" subs="+quotes,attributes"]
+----
+{certs-generate} --foreman-proxy-fqdn "{smartproxy-example-com}" \
+--certs-tar  "_/root/_My_Certificates_/{smartproxy-example-com}-certs.tar_" \
+--server-cert "_/root/_My_Certificates_/New-{smart-proxy-context}_cert.pem_" \
+--server-key "_/root/_My_Certificates_/{smart-proxy-context}_cert_key.pem_" \
+--server-ca-cert "_/root/_My_Certificates_/ca_cert_bundle.pem_" \
+--certs-update-server
+----
+. On {ProjectServer}, copy the certificate archive file to your {SmartProxyServer}:
++
+[options="nowrap" subs="+quotes,attributes"]
+----
+# scp _/root/{smart-proxy-context}_cert/{smartproxy-example-com}-certs.tar_ \
+----
+. Retain a copy of the `{foreman-installer}` command that the `{certs-generate}` command returns for deploying the certificate to your {SmartProxyServer}.
+. Deploy the certificate on your {SmartProxyServer} using the `{foreman-installer}` command returned by the `{certs-generate}` command:
++
+[options="nowrap" subs="+quotes,attributes"]
+----
+# {installer-scenario-smartproxy} \
+--foreman-proxy-content-parent-fqdn "{foreman-example-com}" \
+--foreman-proxy-register-in-foreman "true" \
+--foreman-proxy-foreman-base-url \ "https://{foreman-example-com}"
+--foreman-proxy-trusted-hosts \ "{foreman-example-com}" \
+--foreman-proxy-trusted-hosts "{smartproxy-example-com}" \
+--foreman-proxy-oauth-consumer-key "_My_OAuth_Consumer_Key_" \
+--foreman-proxy-oauth-consumer-secret "_My_OAuth_Consumer_Secret_" \
+--foreman-proxy-content-certs-tar "_/root/{smartproxy-example-com}-certs.tar_" \
+--puppet-server-foreman-url "https://{foreman-example-com}" \
+--certs-update-server
+----
++
+When network connections or ports to {Project} are not yet open, you can set the `--foreman-proxy-register-in-foreman` option to _false_ to prevent {SmartProxy} from attempting to connect to {Project} and reporting errors.
+Run the installer again with the above option set to _true_ when the network and firewalls are correctly configured.
+
+[IMPORTANT]
+====
+Do not delete the certificate archive file after you deploy the certificate.
+It is required, for example, when upgrading {SmartProxyServer}.
```

Comment:
The wording is slightly different than above. I like this one better.

Feel free to either copy&paste; or create a small snippet with the IMPORTANT admonition like this one: https://raw.githubusercontent.com/theforeman/foreman-documentation/master/guides/common/modules/snip_do-not-attach-subscriptions-to-key.adoc

Reply:
I didn't catch about changes in wording. 🙄
I can definitely make a snippet for this, though.

Reply:
Applied the style from `proxy` to `project` section. :+1: